rusk-abi/src/lib.rs

```diff
@@ -57,6 +63,24 @@ pub enum PublicInput {
     JubJubScalar(JubJubScalar),
 }
 
+impl From<BlsScalar> for PublicInput {
+    fn from(s: BlsScalar) -> PublicInput {
+        Self::BlsScalar(s)
+    }
+}
+
+impl From<JubJubScalar> for PublicInput {
+    fn from(s: JubJubScalar) -> PublicInput {
+        Self::JubJubScalar(s)
+    }
+}
+
+impl From<JubJubAffine> for PublicInput {
+    fn from(p: JubJubAffine) -> PublicInput {
+        Self::Point(p)
+    }
+}
+
```

Comment:
If you're going to add this, maybe it would be better having a separate module for `PublicInput`.

Reply:
Do you think its worthy to create a module for something that is ephemeral and should last only until the next release of plonk?

Reply:
Definitely, yes. I don't want the main lib.rs pollute with temporary things, I prefer to have it isolated if the code grows.

To be clear, I agreed with you that we have to implement the `From` traits, so that from the contracts point of view the changes are minimal when we do the proper thing.

However, since this include more temporary code, I want to be sure we isolate it properly.